var/spack/repos/builtin/packages/g4vg/package.py

```diff
@@ -9,12 +9,16 @@ class G4vg(CMakePackage):
     """Generate VecGeom geometry representations from in-memory Geant4 geometry."""
 
     homepage = "https://github.com/celeritas-project/g4vg"
+    git = "https://github.com/celeritas-project/g4vg.git"
     url = "https://github.com/celeritas-project/g4vg/releases/download/v1.0.1/g4vg-1.0.1.tar.gz"
 
     maintainers("sethrj", "drbenmorgan")
 
     license("Apache-2.0", checked_by="sethrj")
 
+    version("develop", branch="main", get_full_repo=True)
+    next_develop_version = "1.1.0"
```

Comment:
Minor comment - is the `next_develop_version` still needed here?

Reply:
No indeed. Thanks!